frontend/src/pages/CalendarPage/components/EventCard/index.js

```diff
@@ -30,6 +31,26 @@ function EventCard({ event, onClose, classes }) {
             <RoomIcon color='disabled' />
             <GetLocation location={event.location} />
           </Box>
+          <Grid>
+            {OfficerRenderPermission(Button)({
+              secondary: true,
+              disabled: event.status === 'pending',
+              children: 'Pending',
+              onClick: () => { const newEvent = { ...event, status: 'pending'}; updateStatus(newEvent, 'pending');},
```

Comment:
Can reduce all of this to a simple updateStatus call, see comment in calendar page.

This applies for the other status buttons as well